dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPTests.java

```diff
@@ -170,7 +168,7 @@ public void testEL30MethodNotFoundException() throws Exception {
         assertEquals("Expected " + 500 + " status code was not returned!",
                      500, response.getResponseCode());
 
-        assertTrue("The response did not contain: javax.el.MethodNotFoundException", response.getText().contains("javax.el.MethodNotFoundException"));
+        assertTrue("The response did not contain: javax.el.MethodNotFoundException", response.getText().contains("el.MethodNotFoundException"));
```

Comment:
Can we do anything with the transformer here so we don't need to remove the javax? In addition we should clean up the assert messages if we're changing what we're actually looking for.